public/_headers

```diff
@@ -2,6 +2,7 @@
   X-Frame-Options: DENY
   X-XSS-Protection: 1; mode=block
   Content-Security-Policy: frame-ancestors 'self'
+  Access-Control-Allow-Origin: *
```

Comment:
is that needed?

Reply:
it seems so

Reply:
noooooo

Reply:
so we can remove this, right?

Reply:
Maybe you can give a better explanation than "nooooo" 😅 

I will try